optuna/structs.py

```diff
@@ -214,23 +212,14 @@ def _validate(self):
     @property
     def distributions(self):
         # type: () -> Dict[str, BaseDistribution]
-        """Return the distributions for this trial.
-
-        Returns:
-            The distributions.
+        """Dictionary that contains the distributions of :attr:`params`.
         """
```

Comment:
This docstring can be a single line.

> flake8 .
> ./optuna/structs.py:215:1: H405: multi line docstring summary not separated with an empty line
> ./optuna/structs.py:216:1: H405: multi line docstring summary not separated with an empty line
> 2     H405: multi line docstring summary not separated with an empty line
https://circleci.com/gh/optuna/optuna/34239?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

```suggestion
        """Dictionary that contains the distributions of :attr:`params`."""
```

Reply:
Applied!